pkg/manager/manager_tasks.go

```diff
@@ -112,7 +114,15 @@ func (s *singleton) Scan(useFileMetadata bool) {
 		total := len(results)
 		logger.Infof("Starting scan of %d files. %d New files found", total, s.neededScan(results))
 
-		var wg sync.WaitGroup
+		start := time.Now()
+		parallelTasks := config.GetParallelTasks()
+		if parallelTasks <= 0 {
+			parallelTasks = runtime.NumCPU()
```

Comment:
since ffmpeg uses multiple threads by default when encoding video ( we limit it to 4 threads for the marker,chunk creation) is it wise to use that many tasks ? 
also the default configuration is `1` shouldn't we use that also here ?

Reply:
While i haven't an issue running that many tasks, i agree for most if won't provide any benefit.

How about 
`parallelTasks` = (runtime.NumCPU() / 4) + 1`

Reply:
I tried it with 4 parallel tasks and the scan process was using ~70% of my server cpu ( dual xeon total 12c/24t). I usually avoid to utilise it that much cpu due to power / noise considerations :-)
Anyway it shouldn't  matter that much ( /4 +1 sounds reasonable to me) since the default is 1 and the user can choose what he wants. Maybe add some more info in the manual  ( https://github.com/stashapp/stash/tree/develop/ui/v2.5/src/docs/en )? Task or Configuration section should be ok.

If you perform tasks remotely (gdrive) insn't it a tradeoff between latency/bw causing cpu underutilisation ( low or 1 parallel task ) and invalidating the cache with many concurrent downloads ( ++ parallel tasks , not sure how rclone handles that though ) ?

Reply:
Resolved as agreed.

Rclone, has three caching modes. 
1. In memory cache. -- Not really caching, but it does do read ahead. 
2. Block based disk cache. -- Good for streaming but can't handle the load of scanning/generation. 
3. File based disk cache. -- Good for scanning but too much latency for browsing/streaming.

I run a large file based cache for scanning (240gb), so it can fit many videos all being scanned/generated concurrently. And i use block based for browsing/streaming.